README.md

```diff
@@ -308,34 +308,14 @@ For example to customize the SQS client:
 ```yaml
 services:
     bref.messenger.sqs_client:
-        class: Aws\Sqs\SqsClient
+        class: AsyncAws\Sqs\SqsClient
         public: true # the AWS clients must be public
         arguments:
             # Apply your own config here
-            -   version: latest
+            -
                 region: us-east-1
 ```
 
-A common use case is mocking clients for tests:
-
-```yaml
-services:
-    bref.messenger.sqs_client:
-        class: Aws\Sqs\SqsClient
-        public: true
-        arguments:
-            -   version: latest
-                region: us-east-1
-                credentials: false
-                # Mock Guzzle for the tests
-                handler: '@mock_handler'
-    mock_handler:
-        class: Aws\MockHandler
-        public: true
-```
-
-You can read more [in the official documentation of the SDK](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_handlers-and-middleware.html).
```

Comment:
How can users mock the client for their tests? Is that possible?

Reply:
They just mock it as a normal class. There is nothing special or fancy like the official SDK. 

See https://github.com/brefphp/symfony-messenger/pull/25/files#diff-32d8454b70ebd9497ea61ebde7ffc540R40-R58

Reply:
Understood, thanks!